scripts/build_demand_profiles.py

```diff
@@ -187,6 +257,7 @@ def build_demand_profiles(
     start_date,
     end_date,
     out_path,
+    file_path_temba,  # fulmicotone
```

Comment:
Best would be to add `file_path_temba` to Snakemake [inputs](https://github.com/pypsa-meets-earth/pypsa-earth/blob/e52848ffc5383b25d92e2768022180175f2ec31f/Snakefile#L420-L428) for `build_demand_profile` rule. It allows for more flexible and clean file management